lumicks/pylake/tests/test_image.py

```diff
@@ -175,3 +175,20 @@ def test_histogram_rows():
         np.testing.assert_allclose(e, [0.0, 0.5])
         assert np.all(np.equal(h, [435, 195]))
         np.testing.assert_allclose(w, [0.5, 0.1])
+
+
+def test_partial_pixel_image_reconstruction():
+    """This function tests whether a partial pixel at the end is fully dropped. This is important,
+    since in the timestamp reconstruction, we subtract the minimum value from a row prior to
+    averaging (to allow taking averages of larger chunks). Without this functionality, the lowest
+    pixel to be reconstructed can be smaller than the first timestamp, which means the subtraction
+    of the minimum is rendered less effective (leading to unnecessarily long reconstruction
+    times)."""
```

Comment:
Should've been lowest `timestamp`. Would've been less confusing I think. How about:
```    """This function tests whether a partial pixel at the end is fully dropped. This is important,
    since in the timestamp reconstruction, we subtract the minimum value from a row prior to
    averaging (to allow taking averages of larger chunks). Without this functionality, the lowest
    timestamp to be reconstructed can be smaller than the first timestamp. This means that the value
    subtracted from the timestamps prior to summing is smaller. This means that the timestamps
    more quickly get into the range where they are at risk of overflow (and therefore have to be
    summed in smaller blocks). This in turn can lead to unnecessarily long reconstruction times."""
```